extensions/wikia/AdEngine/js/provider/gpt/helper.js

```diff
@@ -76,9 +76,8 @@ define('ext.wikia.adEngine.provider.gpt.helper', [
 				slotTargeting.rv = count.toString();
 			}
 		}
-		if (uapId) {
-			slotTargeting.uap = uapId.toString();
-		}
+
+		slotTargeting.uap = uapId ? uapId.toString() : 'none';
```

Comment:
Yep, @fraszczakszymon you're our last resort. We were looking for a way to "reset" slot targeting with pure GPT but it seems there is no way. And resetting it by setting it to falsy values results in our wrapper and implementation working correctly but the request to DFP includes the old `uap` on consecutive PV in Mercury :/